README.md

```diff
@@ -38,6 +38,7 @@ The following tasks are no longer being developed and will be deprecated in a fu
 | Task Name | Alternative |
 |-----------|-------------|
 | st0236_set_cache_paths_to_default | Use [puppet conf](https://forge.puppet.com/modules/puppetlabs/puppet_conf/readme) |
+| st0267_clear_file_sync_locks | statedir can be changed [manually](https://www.puppet.com/docs/puppet/7/configuration.html#statedir) |
```

Comment:
What does changing the `statedir` have to do with this task?  This is about removing stale `index.lock` files that prevent file sync from working.  For an alternative, we need to either list the manual steps from the [kb article](https://perforce1--uat.sandbox.my.site.com/Community2/s/article/360003883933) or link to it.  That still references the task, so we also need to change the article before linking to it.

Reply:
I've put the wrong resolution message in from the documentation - I'll correct that now

Reply:
https://perforce.atlassian.net/browse/SUP-4623

resolved, the KB article will need updated - Ticket created